test-utils/src/commonTest/kotlin/KoinUIThreadViewModelTestTest.kt

```diff
@@ -59,6 +60,7 @@ class KoinUIThreadViewModelTestTest :
         { MyKoinViewModelTestContext() }
 
     @Test
+    @Ignore // Ignored due to a bug in Koin library using Kotlin version >= 1.6.0 https://github.com/InsertKoinIO/koin/issues/1188
```

Comment:
Koin was just updated to version 3.2 which should fix this issue according to the thread. So remove this ignore and track it :)

Reply:
Koin 3.2 is not released yet. I just opened [this issue](https://github.com/splendo/kaluga/issues/460) in order to track this.